e2e-testing/README.md

```diff
@@ -16,3 +16,33 @@ In another terminal, run
 ```
 pnpm build && pnpm test
 ```
+
+#### Quickest way to reset the network/clear all Indexer data sources without rebuilding from scratch
```

Comment:
maybe as a followup we can file a task to write a script to do all of this? Is the network/indexer reset in the e2e tests right now?

Reply:
created https://linear.app/dydx/issue/IND-551/write-script-for-resetting-the-networkclearing-all-indexer-data to track